webserver/views/api/v1/core.py

```diff
@@ -176,35 +189,45 @@ def get_many_lowlevel():
     If the list of MBIDs in the query string has a recording which is not
     present in the database, then it is silently ignored and will not appear
     in the returned data.
- 
+
     :query recording_ids: *Required.* A list of recording MBIDs to retrieve
 
       Takes the form `mbid[:offset];mbid[:offset]`. Offsets are optional, and should
       be >= 0
 
     :resheader Content-Type: *application/json*
-   """
-    recording_ids = request.args.get("recording_ids")
-
-    if not recording_ids:
-        raise webserver.views.api.exceptions.APIBadRequest("Missing `recording_ids` parameter")
-
-    recordings = _parse_bulk_params(recording_ids)
-    if len(recordings) > 200:
-        raise webserver.views.api.exceptions.APIBadRequest("More than 200 recordings not allowed per request")
+    """
+    recordings = check_bad_request_for_multiple_recordings()
 
     recording_details = {}
 
-    # TODO: This should use a bulk get method for speed
     for recording_id, offset in recordings:
         try:
-            recording_details.setdefault(recording_id, {})[offset] = db.data.load_low_level(recording_id, offset)
+            recording_details.setdefault(recording_id, {})[offset] = collect_data(recording_id, offset)
         except NoDataFoundException:
             pass
 
     return jsonify(recording_details)
 
 
+@bp_core.route("/low-level", methods=["GET"])
+@crossdomain()
+def get_many_lowlevel():
+    """Get low-level data for many recordings at once.
```

Comment:
The previous docstring that was here was used to automatically generate documentation on readthedocs for the endpoint. We should keep it and add a similar docstring for the new endpoint added.

Reply:
https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/views/api/v1/core.py# and https://acousticbrainz.readthedocs.io/api.html#get--api-v1-low-level